examples/plotting/rgb_composite.py

```diff
@@ -0,0 +1,58 @@
+"""
+=============================
+Making an RGB composite image
+=============================
+
+This example shows the process required to create an RGB composite image
+of three AIA images at different wavelengths. To read more about the
+algorithm used in this example, see this
+`Astropy tutorial <https://docs.astropy.org/en/stable/visualization/rgb.html>`_.
+"""
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
+from sunkit_image.enhance import mgn
```

Comment:
You'll need to add `sunkit_image` to the documentation requirements for this to work - currently the example is broken: https://sunpy--6459.org.readthedocs.build/en/6459/generated/gallery/plotting/rgb_composite.html#sphx-glr-generated-gallery-plotting-rgb-composite-py

Reply:
Are these requirements located somewhere in the conf.py file in the docs folder?

Reply:
Adding it should work: https://github.com/sunpy/sunpy/blob/main/setup.cfg#L95